README.md

```diff
@@ -28,6 +30,103 @@ git clone git@github.com:kamp-us/monorepo.git
 cd monorepo
 ```
 
+- Install dependencies.
+
+```sh
+npm install
+```
+
+### 2) Modifying `hosts` file
+
+- Add the following text block to [your `/etc/hosts` file](https://www.howtogeek.com/27350/beginner-geek-how-to-edit-your-hosts-file/).
+
+```text
+127.0.0.1 localhost.kamp.us
+127.0.0.1 pasaport.localhost.kamp.us
+127.0.0.1 pano.localhost.kamp.us
+127.0.0.1 sozluk.localhost.kamp.us
+127.0.0.1 gql.localhost.kamp.us
+```
+
+### 3) Setting up `.env` files
+
+- Duplicate `.env.example` files and rename them as `.env` in the following folders: `db/prisma`, `apps/gql`, `apps/kampus`, `apps/pasaport`.
+
+### 4) Prisma Setup and Database Configuration
+
+- Build required packages(`sozluk` and `prisma client`). (It may fail, but it's fine, continue with the next step.)
+
+```sh
+npx turbo build
```

Comment:
This step is no longer needed. when you start throught "npm run dev" this will be run as dependency